client/asset/btc/spv_wrapper.go

```diff
@@ -243,8 +251,6 @@ type spvWallet struct {
 	cfg         *WalletConfig
 	wallet      BTCWallet
 	cl          SPVService
-	acctNum     uint32
-	acctName    string
 	dir         string
```

Comment:
Can we just populate these during initialization with a single call to `AccountInfo` instead of calling it every time?

Reply:
Uhmm, okay.

Reply:
account renaming*

Reply:
Yes, thank you.